File: Components/WinForms/VB/SchedulerMainDemo/MedicData/Patient.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel.DataAnnotations

Namespace DevExpress.XtraScheduler.Demos
	Public Class Patient
		<Key> _
		Public Property Id() As Int64
		Public Property Name() As String
		Public Property Birthday() As DateTime
		Public Property Phone() As String
	End Class
End Namespace

File: Components/WinForms/CS/SchedulerMainDemo/MedicData/Patient.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace DevExpress.XtraScheduler.Demos {
    public class Patient {
        [Key]
        public Int64 Id { get; set; }
        public string Name { get; set; }
        public DateTime Birthday { get; set; }
        public string Phone { get; set; }
    }
}
